### ISSUE TITLE
React Native 0.19 problem with props

### ISSUE THREAD

**Issue by lopezjurip** [AlbertBrand/react-native-android-tablayout#11]
Updated to RN 0.19 and `<Tab/>` props are not working:

``` js
import { Tab, TabLayout } from 'react-native-android-tablayout'
import Colors from '../../global/colors'
```

``` js
<TabLayout
  style={styles.tabs}
  selectedTabIndicatorColor="white"
  selectedTab={this.state.selected}
  onTabSelected={e => this.setState({ selected: e.nativeEvent.position })}>

  <Tab name="Mapas" accessibilityLabel="Mapas" textColor={this.state.selected === 0 ? 'white' :'#d3d3d3'} />
  <Tab name="Información" accessibilityLabel="Información" textColor={this.state.selected === 1 ? 'white' :'#d3d3d3'} />

</TabLayout>
```

``` js
const styles = StyleSheet.create({
  tabs: {
    elevation: 10,
    backgroundColor: Colors.MAIN,
  },
})
```

![image](https://cloud.githubusercontent.com/assets/7570744/12718466/93d6036a-c8cd-11e5-89a7-6499e9caef6d.png)

Expected tabs with white text and white indicator

**Comment by Overload119** [comment]
+1 I have the same issue.

**Comment by AlbertBrand** [comment]
Confirmed. I'll have a look tomorrow.

**Comment by AlbertBrand** [comment]
This is caused by the breaking package name change mentioned in https://github.com/facebook/react-native/issues/5649

**Comment by AlbertBrand** [comment]
Fixed in 0.1.0, which is now RN 0.19+ only.
